Explicit disclosure feature documentation (for the Ledger API) [DPP-1237] (#16119)
digital-asset/daml: docs/source/app-dev/services.rst

For details on how to use command deduplication, see the :doc:`Command Deduplication Guide <command-deduplication>`.

.. _command-explicit-contract-disclosure:

> (Experimental) Explicit contract disclosure

Reviewer: ```suggestion
Explicit contract disclosure (experimental)
```

> Explicit contract disclosure (experimental)

Reviewer 2: ```suggestion
Explicit Contract Disclosure (Experimental)
```

-------------------------------------------

Starting with Canton 2.6, Ledger API clients can use explicit contract disclosure to submit commands with attached

> disclosed contracts that bypass the contract's visibility restriction to stakeholders. For more details,

Reviewer 3: ```suggestion
disclosed contracts received from third-parties. For more details,
```

> Starting with Canton 2.6, Ledger API clients can make use of explicit contract disclosure and submit commands with attached
> disclosed contracts for bypassing the contract's visibility restriction to stakeholders. For more details,
> see :doc:`Explicit contract disclosure <explicit-contract-disclosure>`.

Reviewer: ```suggestion
Starting with Canton 2.6, Ledger API clients can use explicit contract disclosure to submit commands with attached
disclosed contracts that bypass the contract's visibility restriction to stakeholders. For more details,
see :doc:`Explicit contract disclosure <explicit-contract-disclosure>`.
```
